tools/schemacode/schema.py

```diff
@@ -322,3 +359,140 @@ def make_entity_table(schema, tablefmt="github", **kwargs):
     # Print it as markdown
     table_str = tabulate(table, headers="keys", tablefmt=tablefmt)
     return table_str
+
+
+def _get_link(string):
+    dict_ = {
+        "array": "[array](https://www.w3schools.com/js/js_json_arrays.asp)",
+        "arrays": "[arrays](https://www.w3schools.com/js/js_json_arrays.asp)",
+        "string": (
+            "[string](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "strings": (
+            "[strings](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "number": (
+            "[number](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "numbers": (
+            "[numbers](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "object": "[object](https://www.json.org/json-en.html)",
+        "objects": "[objects](https://www.json.org/json-en.html)",
+        "integer": (
+            "[integer](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "integers": (
+            "[integers](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "boolean": (
+            "[boolean](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+        "booleans": (
+            "[booleans](https://www.w3schools.com/js/js_json_datatypes.asp)"
+        ),
+    }
+    string = dict_.get(string, string)
+    return string
+
+
+def _resolve_metadata_type(definition):
+    """Generate string of metadata type from dictionary."""
+    if "type" in definition.keys():
+        string = _get_link(definition["type"])
+
+        if definition.get("enum") == ["n/a"]:
+            # Special string case of n/a
+            string = '`"n/a"`'
+
+        elif "type" in definition.get("items", {}):
+            # Items within arrays
+            string += " of " + _get_link(definition["items"]["type"] + "s")
+
+        elif "type" in definition.get("additionalProperties", {}):
+            # Values within objects
+            string += " of " + _get_link(
+                definition["additionalProperties"]["type"] + "s"
+            )
+
+    elif "anyOf" in definition:
+        substrings = {_resolve_metadata_type(subdict)
+                      for subdict in definition["anyOf"]}
+
+        string = " or ".join(sorted(substrings))
```

Comment:
Would avoid sorting. "array of strings or string" reads more ambiguously than "string or array of strings". Alternately, we can come up with other sorting rules, but I think allowing them to follow the YAML ordering will be least surprising.

```suggestion
        string = " or ".join(substrings)
```